internal/server/server.go

```diff
@@ -169,8 +139,3 @@ func (s *Server) Shutdown(err error) {
 		level.Error(s.logger).Log("msg", "shutting down failed", "err", err)
 	}
 }
-
-func registerProber(r *chi.Mux, p *prober.Prober) {
-	r.Get("/-/healthy", p.HealthyHandler())
```

Comment:
I think we need to update here - https://github.com/observatorium/observatorium/blob/7470f0a697f459ec7db7ec7e73de0dbd638cfff2/jsonnet/lib/observatorium-api.libsonnet as well since we removed the `prober`. right now, I am using the `quay.io/observatorium/observatorium:latest`, the pod cannot be running.
```
  Warning  Unhealthy  52m (x3 over 53m)      kubelet, ip-10-0-137-85.us-east-2.compute.internal  Liveness probe failed: HTTP probe failed with statuscode: 404
  Warning  BackOff    8m16s (x28 over 21m)   kubelet, ip-10-0-137-85.us-east-2.compute.internal  Back-off restarting failed container
  Warning  Unhealthy  3m12s (x475 over 53m)  kubelet, ip-10-0-137-85.us-east-2.compute.internal  Readiness probe failed: HTTP probe failed with statuscode: 404
```

Reply:
Yes, that needs updating. I'm surprised people use `:latest` though. Best example why not to use `:latest`. :wink: